### ISSUE TITLE
--port option not working

### ISSUE THREAD

**Issue by hullabaloooooooo** [mockoon/mockoon#1308]
I'm trying to start mockoon-cli at specific port. But --port option ignoring 

*Mockoon version:6.2.0*

First time I tried to start docker-compose with mockoon service.

```yaml
    mockoon:
        image: mockoon/cli:latest
        command: ["--data", "./data1.json", "./data2.json", "--port", "3001", "3002"]
        networks:
            - sail
        ports:
            - "1080:3001"
            - "1081:3002"
        volumes:
            - './tests/mockoon/carworld_eye.json:/data1.json'
            - './tests/mockoon/yandex_cloud_text.json:/data2.json'
``` 
But all time I got an error - Port 3000 already in use.

Next time I started service with only one data file, then inside container run
```bash mockoon-cli start --data data2.json --port 5001 ```
And same error here
 
<img width="790" alt="image" src="https://github.com/mockoon/mockoon/assets/20566303/db955ce0-e93c-4b8d-ab17-64c1dae8c569">

**Comment by hullabaloooooooo** [comment]
Seems like you miss **_port_** as second param here:

https://github.com/mockoon/mockoon/blob/fe2ced193c2bfbd7fe1ec1ebab42a9aa582b6d3b/packages/cli/src/libs/data.ts#L85

**Comment by 255kb** [comment]
📦 This is now fixed in [v7.0.0](https://mockoon.com/releases/7.0.0/).
